types/lodash/common/object.d.ts

```diff
@@ -1897,20 +1897,20 @@ declare module "../index" {
          */
         merge(...otherArgs: any[]): ObjectChain<any>;
     }
-    type MergeWithCustomizer = { bivariantHack(value: any, srcValue: any, key: string, object: any, source: any): any; }["bivariantHack"];
+    type MergeWithCustomizer = { bivariantHack(value: any, srcValue: any, key: string, object: any, source: any, stack: any): any; }["bivariantHack"];
     // TODO: Probably should just put all these methods on Object and forget about it.
     // oh, except for Collection<any> I GUESS
     interface LoDashStatic {
         /**
          * This method is like `_.merge` except that it accepts `customizer` which
          * is invoked to produce the merged values of the destination and source
          * properties. If `customizer` returns `undefined` merging is handled by the
-         * method instead. The `customizer` is invoked with seven arguments:
+         * method instead. The `customizer` is invoked with six arguments:
          * (objValue, srcValue, key, object, source, stack).
          *
          * @category Object
          * @param object The destination object.
-         * @param sources The source objects.
+         * @param source The source objects.
```

Comment:
The `sources` doesn't match the `source` argument name of the `mergeWith` function:
![image](https://github.com/user-attachments/assets/6982215f-3e8f-4b05-be09-5e2782fc6d9b)